JSDemos/Demos/Scheduler/GroupOrientation/description.md

```diff
@@ -1 +1,5 @@
-This example shows the **workWeek** view with different group orientations. Select the *"Vertical Grouping"* tab to&nbsp;see the resource headers organized in&nbsp;a&nbsp;column and the *"Horizontal Grouping"*&nbsp;&mdash; arranged in&nbsp;a&nbsp;row.
\ No newline at end of file
+[Resources](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/resources/) are categories used to classify appointments. To understand what resources are, consider the following example: in an educational center lectures are held in several rooms; in Scheduler terms, room is a resource kind, individual rooms are resource instances, and lectures are appointments that use these resource instances.
+
+The [Resources demo](/Demos/WidgetsGallery/Demo/Scheduler/Resources/) describes how to assign resources. When assigned, they can also be used to group appointments. To implement this, specify the [groups[]](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#groups) array. Each element of this array is [fieldExpr](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/resources/#fieldExpr) of a resource kind. The order of resource headers in the UI is the same as the order of the resource instances. Note that if the **resources** array has more than one element, grouping is nested. 
+
+The Scheduler arranges group headers vertically (in a column) or horizontally (in a row). You can use the **views**.[groupOrientation](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#groupOrientation) property to set custom orientation for specific views. This property description includes default view orientation types.
```

Comment:
> The Scheduler arranges

Without "can", it reads as if the Scheduler chooses group orientation randomly.

> This property description includes default view orientation types.

It goes without saying. We better refer to the demo instead.


```suggestion
The Scheduler can arrange group headers vertically (in a column) or horizontally (in a row). You can use the **views**.[groupOrientation](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#groupOrientation) property to set custom orientation for specific views. In this demo, the Work Week view type illustrates the difference between vertical and horizontal group orientations.
```